dashboard/app/models/pd/workshop_constants.rb

```diff
@@ -66,6 +68,16 @@ module WorkshopConstants
         SUBJECT_CSP_TEACHER_CON => {max_hours: 33.5},
         SUBJECT_CSP_FOR_RETURNING_TEACHERS => {max_hours: 7}
       },
+      COURSE_CSA => {
+        SUBJECT_CSA_SUMMER_WORKSHOP => {max_hours: 33.5},
+        SUBJECT_CSA_WORKSHOP_1 => {min_days: 1, max_days: 1, max_hours: 6},
+        SUBJECT_CSA_WORKSHOP_2 => {min_days: 1, max_days: 1, max_hours: 6},
+        SUBJECT_CSA_WORKSHOP_3 => {min_days: 1, max_days: 1, max_hours: 6},
+        SUBJECT_CSA_WORKSHOP_4 => {min_days: 1, max_days: 1, max_hours: 6},
+        SUBJECT_CSA_WORKSHOP_1_2 => {min_days: 2, max_days: 2, max_hours: 12},
+        SUBJECT_CSA_WORKSHOP_3_4 => {min_days: 2, max_days: 2, max_hours: 12},
+        SUBJECT_CSA_TEACHER_CON => {max_hours: 33.5}
```

Comment:
Do we need CSA TeacherCon since we don't run teachercon anymore just summer workshops?

Reply:
maybe not? i don't have any context on teachercon stuff

Reply:
Ya TeacherCons were what we did for local summer workshops before they went local. I think this can probably be removed.